src/runtime_src/core/tools/xbmgmt2/flash/flasher.cpp

```diff
@@ -31,6 +31,12 @@
 #include <boost/algorithm/string.hpp>
 #include "boost/filesystem.hpp"
 
+#ifdef _WIN32
+# pragma warning( disable : 4189 4100 )
+# pragma comment(lib, "Ws2_32.lib")
+/* need to link the lib for the following to work */
+#define le16toh(x) (((((uint16_t)(x)) >> 8))|((((uint16_t)(x)) << 8)))
```

Comment:
Hmm...why are we not using `ntohl()`  as opposed to to creating our own.

Please see:
https://github.com/Xilinx/XRT/blob/master/src/runtime_src/tools/xclbinutil/FDTNode.cxx
Fro more details on what libraries to include and how these functions are used.